### ISSUE TITLE
FLEX_APP_URL_PARK_AN_INTERACTION

### ISSUE THREAD

**Issue by bruno222** [TwilioLatamEngHub/plugin-conversations-park-an-interaction#4]
Readme is not mentioned anything about `FLEX_APP_URL_PARK_AN_INTERACTION` found inside of `.env`

Better say that has to be something like this:

`FLEX_APP_URL_PARK_AN_INTERACTION=https://serverless-park-an-interaction-XXXX-dev.twil.io/park-an-interaction`

**Comment by Jonathan-Dilks-at-Wrisk** [comment]
Agreed, I had to read the serverless code to realise it wanted the whole URL of a single endpoint (bit unusual as most other plugins just require the domain & scheme for multiple endpoints within the serverless service e.g. `https://serverless-park-an-interaction-XXXX-dev.twil.io`)
